doc/source/scripting.rst

```diff
@@ -28,29 +28,25 @@ Papis defines environment variables such as ``PAPIS_LIB`` so that external
 scripts can make use of the user input.
 
 To use the script you can put it somewhere in your ``PATH`` or alternatively
-inside the ``~/.papis/scripts`` folder. If this is the case then you can run
+inside the ``~/.papis/scripts`` folder. In the latter case when you run
 
 ::
 
     papis -h
 
-and you will see that there is another command besides the default
-called ``mail``. In fact, you will see
+you will see that there is another command besides the default commands called
+``mail``:
 
-.. code::
-
-    positional arguments:
-      command               For further information for every command, type in 'papis <command> -h'
-        add                 Add a document into a given library
-        .............       ..........................
-        mail                Email a paper to my friend
```

Comment:
This CLI output looks outdated, I produced a new, accurate example.

Reply:
That's great! Thanks!